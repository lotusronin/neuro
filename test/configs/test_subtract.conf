conf = {
'src' : 'test_subtract.nro',
'expected' : 'test_subtract.expected'
}
