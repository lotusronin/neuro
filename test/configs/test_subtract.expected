5 - 4 is...?
1
